src/SUMMARY.md

```diff
@@ -8,6 +8,7 @@
     - [Downloading Images](./installation/liveimages/downloading.md)
     - [Preparing Installation Media](./installation/liveimages/prep.md)
     - [Installation Guide](./installation/liveimages/guide.md)
+    - [Installation Guide (chroot)](./installation/liveimages/guide-chroot.md)
```

Comment:
I think this would be better in the Guides section.

Reply:
Good idea. Will move.